src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -98,6 +98,11 @@ public void ConfigureServices(IServiceCollection services)
         {
             exprLogger.LogExperimentMessage();
 
+            foreach (string warning in listenResults.Warnings)
+            {
+                logger.LogWarning(warning);
```

Comment:
This should be moved to `LoggingExtensions` so that we can give it an `EventId`; maybe call it `UpgradeMetricsUrl`.